it/controllers/employment/SingleEmploymentSummaryEOYControllerISpec.scala

```diff
@@ -80,9 +80,9 @@ class SingleEmploymentSummaryEOYControllerISpec extends IntegrationTest with Vie
     val doYouNeedAnother: String = "Do you need to add another employment?"
     val yesText: String = "Yes"
     val noText: String = "No"
-    val change: String = "Change"
-    val remove: String = "Remove"
     val name: String = "Maggie"
+    val change: String = s"Change Change employment information for $name"
+    val remove: String = s"Remove Remove employment information for $name"
```

Comment:
Change and remove seem to be there twice which looks odd

Reply:
It's because the selector is for the link, text and hidden text. The first change is for the text on the link and the second change part of the hidden text. This is the same with the removes

Reply:
I've added comments in the tests for clarity